app/views/subnets/_fields.html.erb

```diff
@@ -5,7 +5,16 @@
 <%= text_f f, :dns_primary, :help_inline => _("Optional: Primary DNS for this subnet") %>
 <%= text_f f, :dns_secondary, :help_inline => _("Optional: Secondary DNS for this subnet") %>
 <%= selectable_f f, :ipam, Subnet.ipam_modes_with_translations, {}, :data => {'disable-auto-suggest-on' => Subnet::IPAM_MODES[:none]},
-                 :label => _('IPAM'), :help_block => _('IP Address auto suggestion mode for this subnet, DHCP works with DHCP proxy only') %>
+                 :label => _('IPAM'),
+                 :help_inline => popover(
+                         _("IP Address Management"),
+                         _("You can select one of three possible IPAM modes:<br/>" +
+                                   "<ul><li><strong>DHCP</strong> - will manage the IP on DHCP through assigned DHCP proxy, auto-suggested IPs come from DHCP</li>" +
```

Comment:
This text might be a bit nightmarish to translate with so many labels - but I can't think of anything better now, @domcleal maybe you want to follow up this PR with something?